scripts/prep-spd-roster.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+import sys
+
+from pandas.core.frame import DataFrame
+
+
+def main():
+    date = sys.argv[1]
+    in_csv = sys.argv[2]
+    out_csv = sys.argv[3]
+
+    # Import CSV passed to script
+    df = pd.read_csv(in_csv)
+
+    # Rename columns to better formatting
+    column_names = {
+        'Name': 'full_name',
+        'Badge_Num': 'badge',
+        'Title_Description': 'title',
+        'Unit': 'unit',
+        'Unit_Description': 'unit_description'
+    }
+    df.rename(columns=column_names, inplace=True)
+
+    # Split the names out
+    df['first_name'] = df.apply(lambda x: first_name(x), axis=1)

```

Comment:
I think this could be `df["full_name"].str.split(" ")[1].str.strip()` instead of calling apply (same for last name below)

Reply:
I wasn't able to put anything together that worked properly. I could get the string split but I couldn't reference the specific element of the list to return just a string, and instead was storing the entire list in the new column.

Reply:
Not at my computer right now but mental note to myself to point out where I'm doing something really similar in the SDSA importer tomorrow

Reply:
Whoops sorry I forgot about this. Turns out the missing bit was `.str[1]` instead of just `[1]`:
https://github.com/SeattleDSA/sdsa-member-importer/blob/feature/python/sdsa_member_importer/data_utils.py#L52-L56

Reply:
Oh hell that makes sense. Lemme try it out.